### ISSUE TITLE
Sticky menu - Chrome problem

### ISSUE THREAD

**Issue by pawelszroeder** [pojome/elementor#4810]
Hi, I've noticed that when I use sticky header, position: fixed start to appear in code only when I start to scroll. Because of that in Chrome there is strange jump effect of some pixels with header. In Firefox everything is ok.

Is it possible to apply position: fix when page is loaded?

**Comment by Artneo7** [comment]
When the page is refreshed and before scrolling when you click an anchor link on menu it overlaps the content (as you can see in the image below).

![overlap](https://user-images.githubusercontent.com/17693529/41865008-3209c40c-7882-11e8-9938-e49487bd5c66.png)

As soon as you start scrolling this issue is gone. Maybe @pawelszroeder request will fix this issue.

**Comment by pawelszroeder** [comment]
Yeah, this is the same problem that I have. I wrote to support yesterday and they say that it would be fix in next update.

Sticky header jumps down when I start to scroll.
More over menu item is scrolling to bad position. It is scrolling ok when I click again or when I first time click after make some scrolling with mouse.

I've recorded video with these 2 problems here:
https://www.useloom.com/share/325072332eff485e9ea5a2613b2cbd9e

**Comment by iamlasse** [comment]
This issue is still happening, I just experienced it for a client on Elementor 3. The problem is on first click the custom css added adheres to the top of the page, and on second click, the link/anchor adheres to the bottom of the sticky menu. Thereby adding 2 different offsets and appearing wonky in performance.
